microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -37,3 +37,21 @@ groups:
         description: The percentage of time that the virtual CPU on node {{ $labels.instance }} is waiting for a real CPU is above 3%.
           This can be due to the fact that a virtual Machine is undersized or the hypervisor is overcommited.
         remediation: Increase the resources available to the VM or reduce the number of VMs on the host machine.
+
+    - alert: CB90043-diskCommitFail
+      expr: |
+        kv_ep_data_write_failed > 0
```

Comment:
If I recall correctly, `kv_ep_data_write_failed` is a counter, meaning it only ever goes up, never down - which means that, as soon as you get a single write commit failure, this alert will start firing forever. You may need to wrap it in something like `increase(kv_ep_data_write_failed[24h]) > 0` or something like that.

Reply:
Or maybe something less than 24h, because that means that Prometheus has to cr0nch 24 hours worth of data each time this rule evaluates... but OTOH a write commit failure is serious enough that you'd want it to fire for a while... hmm...

Reply:
I agree, if an alert is meant to be "wake up at 2AM" then having it run for 8-12 hours seems reasonable

Reply:
Looking at various [alert templates](https://awesome-prometheus-alerts.grep.to/rules.html), you can probably get away with having it be as low as `1m` (and set `for: 0m` while you're at it) - the alert will fire anyway, and even if it gets resolved quickly it'll still be logged. How we display that in Grafana is a bit of a complication, since it'll disappear quickly... hmmmmmmmmm

Reply:
I'll set it to that and leave the fallout to you and Dougie :p